File: io/rpcclient.py
# coding: utf-8
import xmlrpc.client

with xmlrpc.client.ServerProxy("http://localhost:8081/") as server:
    result = server.soma(5,3)
    print("O resultado da soma é:", result["resultado"])

    result = server.divide(11,4)
    print("O resultado da divisão é", result["resultado"], "e o resto é", result["resto"])

File: br/io/rpcclient.rb
require "xmlrpc/client"

begin
  client = XMLRPC::Client.new("localhost", "/RPC2", 8081)
  resp = client.call("soma", 5, 3)
  puts "O resultado da soma é #{resp['resultado']}"

  resp = client.call("divide", 11, 4)
  puts "O resultado da divisao é #{resp['resultado']} e o resto é #{resp['resto']}"
rescue => exception
  puts "ERRO: #{exception}"
end
